api/bucketsd/service.go

```diff
@@ -2401,7 +2405,7 @@ func (s *Service) updateOrAddPin(ctx context.Context, from, to path.Path) (conte
 	owner, ok := buckets.BucketOwnerFromContext(ctx)
 	if ok {
 		log.Debugf("updateOrAddPin: storage: %d used, %d available, %d delta",
-			owner.StorageUsed, owner.StorageAvailable, owner.StorageDelta)
+			owner.StorageUsed, owner.StorageAvailable, deltaSize)
```

Comment:
it's reporting some value out of memory which is only some previous delta (usually 0). here, reporting the current incoming delta